package.json

```diff
@@ -53,7 +53,7 @@
     "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-prettier": "^5.2.3",
     "eslint-plugin-react": "^7.37.4",
-    "eslint-plugin-react-hooks": "^4.6.2",
+    "eslint-plugin-react-hooks": "^5.1.0",
```

Comment:
@alimpens We don’t use any rules or configuration of this package. Should we? If so, is it a matter of including `plugin:react-hooks/recommended` into our ESLint config?

Reply:
I think this is used by [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb#eslint-config-airbnb-1). I don't think we're currently using any rules for hooks, to do so we could add `airbnb/hooks` to our `extends` list.